environments/drupal-7.sh

```diff
@@ -5,7 +5,7 @@
 function drupal_ti_install_drupal() {
 	drush --yes dl drupal-7 --drupal-project-rename=drupal
 	cd drupal
-	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install testing --db-url="$DRUPAL_TI_DB_URL"
+	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install $DRUPAL_TI_INSTALL_PROFILE --db-url="$DRUPAL_TI_DB_URL"
```

Comment:
Sorry, been focusing on getting File (Field) Paths a long overdue release,
I will get back to this soon, if not at DrupalCon.

--

http://stuar.tc/lark
@Decipher
On 3 Sep 2015 11:24 pm, "Fabian Franz" <notifications@github.com> wrote:

> In environments/drupal-7.sh
> <https://github.com/LionsAd/drupal_ti/pull/36#discussion_r38644390>:
>
> > @@ -5,7 +5,7 @@
> >  function drupal_ti_install_drupal() {
> >  	drush --yes dl drupal-7 --drupal-project-rename=drupal
> >  	cd drupal
> > -	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install testing --db-url="$DRUPAL_TI_DB_URL"
> > +	php -d sendmail_path=$(which true) ~/.composer/vendor/bin/drush.php --yes site-install $DRUPAL_TI_INSTALL_PROFILE --db-url="$DRUPAL_TI_DB_URL"
>
> Any update?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LionsAd/drupal_ti/pull/36/files#r38644390>.
>